src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,42 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static #animalsList = [];
+
+  constructor(name, health = 100) {
+    this.health = health;
+    this.name = name;
+    Animal.#animalsList.push(this);
+  }
+
+  static get alive() {
+    return Animal.#animalsList.filter((animal) => animal.health > 0);
```

Comment:
The `alive` getter uses `filter` to return only alive animals, which is a good use of array methods and avoids nested loops, meeting the code performance requirement.